### PR TITLE
T244850 scrolldown arrow

### PR DESCRIPTION
**Pull request by hueitan**
Phabricator Link: https://phabricator.wikimedia.org/T244850

### Problem Statement

Readers need to understand that they can scroll down for more content:

### Solution

Adding the icon arrow to show user to scroll for more content

### Note

| Screen Shot 2020-03-10 at 11 35 56 | Screen Shot 2020-03-10 at 11 35 48 |
| --- | --- |

### CODE REVIEW

File: src/components/Article.js [hunk 1]
```diff
@@ -83,7 +83,7 @@ const ArticleSection = ({
 
       const introNode = contentRef.current.querySelector('.intro')
       let introHeight = introNode.getBoundingClientRect().height
-      introHeight += 34 // Magic number needed to make it work
+      introHeight += 21 // Magic number needed to make it work
```
Comment (medied) [inline src/components/Article.js]: Knowing what to set here makes you a magician 🧙‍♂️😯

**Comment by hueitan** [comment]
@jpita ready to be tested

File: src/components/Article.js [hunk 2]
```diff
@@ -71,7 +71,7 @@ const ArticleSection = ({
     if (!contentRef.current) {
       return
     }
-    const MAX_TITLE_HEIGHT = 140
+    const MAX_TITLE_HEIGHT = 170
```
Comment (hueitan) [inline src/components/Article.js]: oppps I shouldn't change this, this is not related here
